UPGRADE.md

```diff
@@ -6,6 +6,6 @@
 * all methods have now return types
 * all constructors now use property promotion
 * all deprecated features were removed
-* the new default type for mapping is "attribute". You can still use annotations, but you need an explicit definition.
+* [BC BREAK] the new default type for mapping is "attribute". You can still use annotations, but you need an explicit definition (set "annotation" as value for vich_uploader.metadata.type config key)
```

Comment:
every single point in this list is a BC break, so please remove the tag.
Moreover, the key config name must be quoted.

Reply:
Done. Thanks for your fast response.